app/src/main/res/values/styles.xml

```diff
@@ -82,8 +82,14 @@
         <item name="windowNoTitle">true</item>
     </style>
 
-    <style name="SplashTheme" parent="@style/AppTheme">
-        <item name="android:windowBackground">@drawable/background_splash</item>
+    <!--    Splash Screen Theme-->
+    <style name="Theme.App.Starting" parent="Theme.SplashScreen">
+
+        <item name="windowSplashScreenBackground">@color/ic_launcher_background</item>
+
+        <item name="windowSplashScreenAnimatedIcon">@drawable/background_splash</item>
```

Comment:
This looks really bad on my Android 12 emulator somehow (Pixel 4)

![Screenshot_1633032023](https://user-images.githubusercontent.com/1885159/135525211-d7dd8c46-8086-4697-bcd1-2997341a8be7.png)

I think you probably shouldn't use the previous complete splash screen as only the icon, but instead use ic_launcher_foreground here? Then we can probably just delete the background_splash file.

Reply:
I have deleted the background_splash file

Reply:
Looks good on my Android 12 emulator and Android 11 physical device now :)